packages/babel-plugin-minify-constant-folding/src/replacements.js

```diff
@@ -32,16 +32,6 @@ module.exports = ({ types: t }) => {
         }
       },
       calls: {
-        concat(...args) {
-          return t.arrayExpression(
-            this.elements.concat(
-              ...args.map(arg => {
-                if (t.isArrayExpression(arg)) return arg.elements;
```

Comment:
Would it be possible to check if any of these are variables instead of removing this entirely? For example, `[a, b, c].concat([1, 2, 3])` is safe.